nightingale_dispatcher/src/nightingale_dispatcher/task_action_server.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+import rospy
+import actionlib
+import time
+
+from actionlib_msgs.msg import GoalStatus
+from nightingale_msgs.msg import TaskAction
+
+
+class TaskActionServer:
```

Comment:
im not sure that this should be an action server. i point back to my comments about having a single process/file